java/athena/src/test/java/sleeper/athena/record/SimpleRecordHandlerIT.java

```diff
@@ -347,11 +344,11 @@ public void shouldNotBringBackValueIfItWasNotAskedFor() throws Exception {
         ));
 
         // Then
-        assertTrue(response instanceof ReadRecordsResponse);
-        assertEquals(28, ((ReadRecordsResponse) response).getRecordCount());
```

Comment:
Looks like this test is flaky. This assertion is failing sometimes, returning 29 instead of 28: https://github.com/gchq/sleeper/runs/7788544389

Reply:
Looks like there's a separate issue for this: https://github.com/gchq/sleeper/issues/104